waku/v2/node/waku_node.nim

```diff
@@ -186,7 +197,8 @@ proc new*(T: type WakuNode,
     sendSignedPeerRecord = sendSignedPeerRecord,
     agentString = agentString,
     peerStoreCapacity = peerStoreCapacity,
-    services = @[Service(getAutonatService(rng))],
+    services = @[Service(hpservice)],
```

Comment:
just to make sure i got it right: autonatservice is included indirectly in `hpservice`? so no need for `services=[hpservice, autonatservice]`?

wondering what would happen if i try:
`services=[hpservice, autonatservice]`

Reply:
Yes, `autonatservice` is added at line 184. Regarding your second question, it'd create two `autonatservice`, I didn't test it tbh, but it shouldn't be done.

Reply:
I see the `hpservice` as a wrapper around `autonatservice`, so I think we don't need to add it twice.

https://github.com/status-im/nim-libp2p/blob/74c402ed9db4652a455c00c8d1713b222e3ef3d5/libp2p/services/hpservice.nim#L110